.github/eosjs-ci/Dockerfile

```diff
@@ -10,52 +10,31 @@ WORKDIR /root
- ; fi
-
-RUN if [ "$CDTBRANCH" = "release_1.8.x" ] || [ "$CDTBRANCH" = "develop" ]; then \
- git clone https://github.com/EOSIO/eos \
+RUN git clone https://github.com/EOSIO/eos \
  && cd eos \
  && git checkout develop \
  && mkdir -p /root/contracts/eosio.bios/ && mv ./contracts/contracts/eosio.bios/bin/* /root/contracts/eosio.bios/ \
  && mkdir -p /root/contracts/eosio.boot/ && mv ./contracts/contracts/eosio.boot/bin/* /root/contracts/eosio.boot/ \
  && mkdir -p /root/contracts/eosio.system/ && mv ./unittests/contracts/eosio.system/* /root/contracts/eosio.system/ \
  && mkdir -p /root/contracts/eosio.msig/ && mv ./unittests/contracts/eosio.msig/* /root/contracts/eosio.msig/ \
- && mkdir -p /root/contracts/eosio.token/ && mv ./unittests/contracts/eosio.token/* /root/contracts/eosio.token/ \
- ; fi
+ && mkdir -p /root/contracts/eosio.token/ && mv ./unittests/contracts/eosio.token/* /root/contracts/eosio.token/
 
-RUN if [ "$CDTBRANCH" = "develop" ]; then \
- git clone https://github.com/EOSIO/eosio.cdt \
+RUN git clone https://github.com/EOSIO/eosio.cdt \
  && cd eosio.cdt \
  && git checkout develop \
  && mkdir -p build/read_only_query_tests/ \
  && eosio-cpp -abigen ./tests/unit/test_contracts/read_only_query_tests.cpp -o ./build/read_only_query_tests/read_only_query_tests.wasm \
- && mkdir -p /root/contracts/read_only_query_tests/ && mv ./build/read_only_query_tests/* /root/contracts/read_only_query_tests/ \
- ; fi
+ && mkdir -p /root/contracts/read_only_query_tests/ && mv ./build/read_only_query_tests/* /root/contracts/read_only_query_tests/
 
-RUN if [ "$CDTBRANCH" = "release_1.8.x" ] || [ "$CDTBRANCH" = "develop" ]; then \
- git clone https://github.com/EOSIO/key-value-example-app.git \
+RUN git clone https://github.com/EOSIO/key-value-example-app.git \
  && cd key-value-example-app \
  && eosio-cpp -abigen ./contracts/kv_todo/src/kv_todo.cpp -o ./contracts/kv_todo/build/kv_todo.wasm -R ./contracts/kv_todo/ricardian/ -I ./contracts/kv_todo/include/ \
- && mkdir -p /root/contracts/kv_todo/ && mv ./contracts/kv_todo/build/* /root/contracts/kv_todo/ \
- ; fi
+ && mkdir -p /root/contracts/kv_todo/ && mv ./contracts/kv_todo/build/* /root/contracts/kv_todo/
 
-RUN if [ "$CDTBRANCH" = "release_1.8.x" ] || [ "$CDTBRANCH" = "develop" ]; then \
- git clone https://github.com/EOSIO/return-values-example-app.git \
+RUN git clone https://github.com/EOSIO/return-values-example-app.git \
  && cd return-values-example-app \
  && eosio-cpp -abigen ./contracts/action_return_value/src/action_return_value.cpp -o ./contracts/action_return_value/build/action_return_value.wasm -R ./contracts/action_return_value/ricardian/ \
- && mkdir -p /root/contracts/action_return_value/ && mv ./contracts/action_return_value/build/* /root/contracts/action_return_value/ \
- ; fi
+ && mkdir -p /root/contracts/action_return_value/ && mv ./contracts/action_return_value/build/* /root/contracts/action_return_value/
```

Comment:
Could we use git fetch to grab a few files in place of cloning whole repos here ?

Reply:
Builds take 2 minutes max at the moment so minimal performance increases for that change.  Will probably address in the future.